readme.md

```diff
@@ -98,3 +98,14 @@ subscribe(state, () => {
   const obj = snapshot(state) // A snapshot is an immutable object
 })
 ```
+
+##### Use it locally in components
+
+You can use it locally in components. [Note](./src/utils.ts#L5-L15)
+
```

Comment:
Now it's great, but can't we put the notes here, I don't have any problem with this, but just a question.

Reply:
I just feel a bit long..

Reply:
That's ok, no problem.